packages/kong-plugin/spec/readme-plugin/01-integration_spec.lua

```diff
@@ -0,0 +1,98 @@
+-- Helper functions provided by Kong Gateway, see https://github.com/Kong/kong/blob/master/spec/helpers.lua
+local helpers = require "spec.helpers"
```

Comment:
whats our integration test environment like? if it can run docker then we should be able to create some! otherwise we won't really be able to run a kong instance.